app/assets/stylesheets/00-css-reset.css

```diff
@@ -46,4 +46,8 @@ q:before, q:after {
 table {
 	border-collapse: collapse;
 	border-spacing: 0;
+}
+
+* {
```

Comment:
What does * do? Also I'm curious why your css files are numbered?

Reply:
`*` selects for all elements. It was recommended to add box-sizing to everything to make layout easier because it will case element width to include border and padding.

My CSS files are numbered because old Rails uses a CSS compiler, so we were originally taught to separate the CSS files by component for organization, but the numbered filenames keep the files in sequence since CSS code in a later file could override previous code, if that makes sense.